App.js

```diff
@@ -26,8 +26,9 @@ import {
 } from 'react-native/Libraries/NewAppScreen';
 
 import axios from 'axios';
-import {Button} from 'react-native-material-ui';
+import {Button, Card} from 'react-native-material-ui';
```

Comment:
are we still using material ui?

Reply:
I'm not. I leave it because I'm not sure if you want to keep the topnav.